src/main/java/Hsien.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Hsien {
+
+    public static void printLogo() {
+        System.out.println(" _   _         _____        _ _   _ ");
+        System.out.println("| | | | ##### |_   _|##### |   \\ | |");
+        System.out.println("| |_| |#        | |  #     | |\\ \\| |");
+        System.out.println("|  _  | #####   | |  ##### | | \\ \\ |");
+        System.out.println("| | | |      # _| |_ #     | |  \\  |");
+        System.out.println("|_| |_| ##### |_____|##### |_|   \\_|\n");
+    }
+
+    public static void printLine() {
+        System.out.println("\n" + "-".repeat(50) + "\n");
+    }
+
+    public static void printList(ArrayList<Task> tasks) {
+        int counter = 1;
+        System.out.println("Here are the tasks in your list!");
+        for (Task t : tasks) {
+            System.out.printf("%s. %s%n", counter, t.getStatusDescription());
+            counter += 1;
+        }
+    }
+
+    public static void main(String[] args) {
+        printLine();
+        printLogo();
+        // Greet
+        System.out.println("Hello! I am Hsien, your personal chatbot");
+        printLine();
+
+        ArrayList<Task> messages = new ArrayList<>();
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            System.out.print("Please enter a commnd/add task: ");
+            String command = in.nextLine();
+            Task newTask = new Task(command);
+            if (command.equals("bye")) {
```

Comment:
the embedded if-else-elseif statements are confusing - is it possible to combine or move some statements elsewhere from this?